### PR TITLE
Created Fethc function with GET, POST, DELETE

### PR DESCRIPTION
**Pull request by raul-2001**
Hello, 
Thanks for reviewing my code.
I have created some fetch data functions (with GET, POST, DELETE).
I know that I have to create a new common function and use this function. Could you explain to me how to do it? 
Thank you so much.

### CODE REVIEW

**Comment by gsiryvhb** [comment]
@raul-2001 please note - I do not get notifications for Pull Requests unless you submit another CTD Review form -

File: src/App.js [hunk 1]
```diff
@@ -7,55 +7,196 @@ import AddTodoForm from './AddTodoForm';
...
 
+    fetchData();
   }, [])
 
+  const addTodo = (records) => {
+
+    // set the todoList for Get
+    setTodoList([...todoList, records[0].fields]);
+    // set the postTodoList for Post 
+    setPostTodoList({...postTodoList, records});
```
Comment (gsiryvhb) [inline src/App.js]: @raul-2001 there is no need to maintain 2 to do lists 

When you add a record, send the post to airtable to add the record, then get your items from airtable again to refresh your list using your fetchData function

**Review by gsiryvhb — CHANGES_REQUESTED** [review]
Please see comments -

**Comment by raul-2001** [comment]
But, I have submitted CTD reveiw form for lesson_1_8.

**Comment by gsiryvhb** [comment]
@raul-2001 I see it - please make the corrections listed above and I will review again

**Comment by raul-2001** [comment]
I made corrections. You can review it. Thanks as well. I apologize that I bother you with these corrections.

**Review by gsiryvhb — APPROVED** [review]
all good

**Pull request merged**